### PR TITLE
chore: add volunteer maniadhikari

### PR DESCRIPTION
**Pull request by maniadhikari**

### CODE REVIEW

File: data/users/maniadhikari.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Manikanta Adhikari",
+  "bio": "| Web Dev | Designer ",
```
Comment (codewithvoid) [inline data/users/maniadhikari.json:3]: It's platform for devs so you can just mention Web Developer :)

File: data/users/maniadhikari.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "image": "https://pbs.twimg.com/profile_images/1606179976159842306/iPN4q7sK_400x400.jpg",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["Web Development", "Design"]
```
Comment (codewithvoid) [inline data/users/maniadhikari.json:8]: Can you mention frameworks and web technologies you use . E.g
```json
"topics": ["HTML", "CSS", "NodeJS"]
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @maniadhikari for volunteering. Can you please check the comments.

File: data/users/maniadhikari.json (hunk 1 above)
Comment (maniadhikari) [reply data/users/maniadhikari.json]: oops! Thanks for letting me know Updated👍

File: data/users/maniadhikari.json (hunk 2 above)
Comment (maniadhikari) [reply data/users/maniadhikari.json]: Sure. mentioned right away.

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes. Welcome @maniadhikari to the community.

**Pull request merged**